run_scripts/2_fetch_raw_data.sh

```diff
@@ -0,0 +1,57 @@
+#!usr/bin/env bash
+
+set -e
+
+if [ $# -ne 5 ]; then
+    echo "usage: ./2_fetch_raw_data.sh <user> <rapid-pro-root> <rapid-pro-server> <rapid-pro-token> <data-root>"
+    echo "Downloads the radio show answers from each show, as well as all the contacts data"
+    exit
+fi 
+
+USER=$1
+RP_DIR=$2
+RP_SERVER=$3
+RP_TOKEN=$4
+DATA_ROOT=$5
+
+TEST_CONTACTS_PATH="$(pwd)/configurations/test_contacts_rapid_pro_ids.json"
+
+SHOWS=(
+    "icraf_s01_e01_activation"
+    "icraf_s01_e02_activation"
+    "icraf_s01_e03_activation"
+    "icraf_s01_e04_activation"
+    "icraf_s01_e05_activation"
+    "icraf_s01_e06_activation"
+)
+
+SURVEYS=(
+    "icraf_s01_demogs"
+    "icraf_s01_fus"
```

Comment:
what is fus?

Reply:
Follow up survey

Reply:
Is it possible to rename the flow to icraf_s01_follow_up_survey?